lib/github.go

```diff
@@ -57,3 +68,76 @@ func PrintableLabels(labels Labels) string {
 
 	return strings.Join(ll[:], ", ")
 }
+
+// GetLabelTypeSet Converts a list of labels into a set of LabelType
+func GetLabelTypeSetFromRepository(repo Repository) LabelTypeSet {
+	labelTypeSet := make(LabelTypeSet)
+	if len(repo.Labels.Nodes) > 0 {
+		for _, node := range repo.Labels.Nodes {
+			labelTypeSet [GetLabelType(node.Name)] = true
+		}
+	}
+
+	return labelTypeSet
+}
+
+// GetLabelTypeSet Converts a list of labels into a set of LabelType
+func GetLabelTypeSet(labels Labels) LabelTypeSet {
+	labelTypeSet := make(LabelTypeSet)
+
+	if len(labels.Edges) > 0 {
+		for _, edge := range labels.Edges {
+			labelTypeSet [GetLabelType(edge.Node.Name)] = true
+		}
+	}
+
+	return labelTypeSet
+}
+
+type LabelTypeSet map[LabelType]bool
+
+type LabelType string
+
+const (
+	UNKNOWN LabelType = "UNKNOWN"
+	NeedsOneReview = "NEEDS ONE REVIEW"
+	NeedsTwoReviews = "NEEDS TWO REVIEWS"
+	READY = "READY"
+	WIP ="WIP"
+)
+
+var oneReviewRegex = regexp.MustCompile(`(?i)needs 1 review`)
```

Comment:
https://golang.org/pkg/strings/#ToLower is faster than using regular expressions